tasksheets/sheet2/sll_grammar.md

```diff
@@ -0,0 +1,142 @@
+# Grammar
+
+* Grammar symbols are generally space-separated.
+* Tokens are generally lower-case or special characters.
+    * `IDENT` is the most notable exception to this rule.
+    * The `(` and `)` tokens are escaped using `\(` and `\)` because they are part of the meta-language as well.
```

Comment:
```suggestion
    * The `(`, `)`, `|` and `*` tokens are escaped with `\`, e.g. `\(`, because they are part of the meta-language as well.
```

Reply:
We only need escaping for (, the others are not part of the object language.
Josh and I were too lazy / it hurts legibility.

Reply:
I wasn't, look at the grammar :smile: